.github/workflows/release-node.yml

```diff
@@ -22,7 +22,7 @@ env:
   RELEASE_TAG: ${{ github.event.inputs.release_tag || github.event.client_payload.release_tag || 'CI_TEST' }}
   APIFY_VERSION: ${{ github.event.inputs.apify_version || github.event.client_payload.apify_version }}
   CRAWLEE_VERSION: ${{ github.event.inputs.crawlee_version || github.event.client_payload.crawlee_version }}
-  NODE_LATEST: 16
+  NODE_LATEST: 18
```

Comment:
@mnmkng @metalwarrior665 wdyt, should we switch the latest tag to node 18?

Reply:
Considering the end of the maintenance cycle for nodejs 16 is approaching, I feel like we should, but would love more thoughts!

![image](https://github.com/apify/apify-actor-docker/assets/17960496/f4ad11e5-44a2-4488-8815-bdc1f1db9041)

Reply:
Yes, I am on the same page but want to see what others think, mainly Lukas, not sure if delivery is using node 18 already, the templates are still on v16 (we should update those for sure regardless of the latest docker tag).

Reply:
Yeah, I'd switch it. We had version tags in Dockerfiles for a long time, so hopefully it should not cause any issues.